charts/kueue/values.yaml

```
@@ -152,6 +152,8 @@ webhookService:
       protocol: TCP
       targetPort: 9443
   type: ClusterIP
+mutatingWebhookConfiguration:
```

```suggestion
mutatingWebhook:
```

Based on the discussion, could you replace this with `mutatingWebhook`?

---

fixed as well